routes/models/proposal.js

```diff
@@ -0,0 +1,132 @@
+var md5= require('md5') ;
+var DB = require('../config.js') ;
+var random = require('../../libs/random.js') ;
+var Boom =  require('boom');
+
+module.exports.add = function(req,res) {
+    var proposal = req.payload ;
+    proposal.created =  new Date();
+    proposal.is_active = 1;
+    DB.conn.queryAsync("INSERT INTO "+DB.proposals +" SET ?" , proposal ).then(function(result)
+    {
+        res({"statusCode":200,"message":"New proposal created successfully"}) ;
+    });
```

Comment:
@tusharncuelogic @sandeepp-cuelogic @gauravc-cuelogic @pritig-cuelogic don't we need to handle exception?